core/src/main/java/org/apache/iceberg/rest/HTTPClient.java

```diff
@@ -78,10 +78,13 @@ public class HTTPClient implements RESTClient {
   static final String CLIENT_GIT_COMMIT_SHORT_HEADER = "X-Client-Git-Commit-Short";
 
   private static final String REST_MAX_RETRIES = "rest.client.max-retries";
-  private static final String REST_MAX_CONNECTIONS = "rest.client.max-connections";
-  private static final int REST_MAX_CONNECTIONS_DEFAULT = 100;
-  private static final String REST_MAX_CONNECTIONS_PER_ROUTE = "rest.client.connections-per-route";
-  private static final int REST_MAX_CONNECTIONS_PER_ROUTE_DEFAULT = 100;
+  @VisibleForTesting static final String REST_MAX_CONNECTIONS = "rest.client.max-connections";
```

Comment:
I don't feel like we need the `@VisibleForTesting` annotations everywhere here.  Just make them package protected.

Reply:
removed the annotation